python/hail/methods/family_methods.py

```diff
@@ -36,13 +37,10 @@ def trio_matrix(dataset, pedigree, complete_trios=False) -> MatrixTable:
     The new column fields consist of three structs (`proband`, `father`,
     `mother`), a Boolean field, and a string field:
 
```

Comment:
There should be a field **id** which is the column key of the matrix. This is from looking at the Scala code and your Python code that uses the trio matrix.

Reply:
this confused me too. I think we should remove the `id` key field in a future PR.